tbx/static_src/sass/config/_variables.scss

```diff
@@ -87,13 +129,22 @@ $small-font-sizes: (
 
 // --------------------------------- Sizing ------------------------------------
 
-$breakpoints: ('medium' '(min-width: 599px)', 'large' '(min-width: 1023px)');
+$breakpoints: (
+    'medium' '(min-width: 599px)',
+    'large' '(min-width: 1023px)',
+    'x-large' '(min-width: 1280px)'
+);
```

Comment:
Note to self: add docs for these

Reply:
Added in https://github.com/torchbox/torchbox.com/pull/10/commits/8d2608e39dd13d56bde60c21055d500be8f16e4a